en/docs/use-cases/streaming-tutorials/expose-a-kafka-topic-as-a-managed-websocket-api.md

```diff
@@ -0,0 +1,391 @@
+
+## Prerequisites
+
+??? note " 1. Configure the Streaming Integrator"
+
+    Follow the steps below to configure the Streaming Integrator to consume the stream from Kafka:
+
+    1. Download the [Kafka broker](https://www.apache.org/dyn/closer.cgi?path=/kafka/2.3.0/kafka_2.12-2.3.0.tgz), which is available on the Apache site and extract it.
+
+        Let's refer to this directory as `<KAFKA_HOME>`.
+
+    2. Copy the required JAR files.
+
+        1. Create a directory named `Source` in a preferred location in your machine.
+        
+        2. Navigate to the `<KAFKA_HOME>/libs` directory and copy the following JARs to the `Source` directory.
+
+            - `kafka_2.12-2.3.0.jar`
+
+            - `kafka-clients-2.3.0.jar`
+
+            - `metrics-core-2.2.0.jar`
+
+            - `scala-library-2.12.8.jar`
+
+            - `zkclient-0.11.jar`
+
+            - `zookeeper-3.4.14.jar`
+
+    3. Create another directory named `Destination` in a preferred location in your machine.
+
+    4. Convert the Kafka JARS into OSGi bundles.
+
+        Execute the following command to convert the Kafka JARS, which are in the `Source` directory.
+
+        ``` bash tab="Windows"
+        <SI_HOME>/bin/jartobundle.bat <source-directory-path> <destination-directory-path>
+        ```
+
+        ``` bash tab="Linux"
+        sh <SI_HOME>/bin/jartobundle.sh <source-directory-path> <destination-directory-path>
+        ```
+
+    5. Add the **OSGi converted Kafka libs** from the `Destination` directory to the `<SI_HOME>/lib` directory.
+
+    6. Add the **original Kafka libs** from `Source` to the `<SI_HOME>/samples/sample-clients/lib` directory.
+
+??? note "2. Start Kafka"
+
+    1.Navigate to the `<KAFKA_HOME>` directory and start a Zookeeper node.
+
+        ```
+        sh bin/zookeeper-server-start.sh config/zookeeper.properties
+        ```
+
+    2. Navigate to the `<KAFKA_HOME>` directory and start the Kafka server node.
+
+        ```
+        sh bin/kafka-server-start.sh config/server.properties
+        ```
+
+??? note "3. Install Apache Ant"
+
+    Download and install [Apache Ant](https://ant.apache.org/).
+
+## Step 1 - Enable publishing to the service catalog
+
+Follow the instructions below to enable publishing the AsyncAPI definition to the service catalog in WSO2 API Manager:
+
+1. Configure the API Manager port.
+
+     You have to define the port to which the Streaming Integrator publishes the AsyncAPI definition.
+
+     1. Open the `<API-M_HOME>/repository/conf/deployment.toml` file.
+
+     2. Uncomment `offset` in the `[server]` section and set it to `5` as shown below.
+
+         ```
+         [server]
+         offset=5
+         ```
+   
+2. Configure the basic details needed to publish to the service catalog.
+
+     1. Open the `<SI_HOME>/conf/server/deployment.yaml` file. 
+     
+     2. Update the `async.api.configs:` section as follows:
+
+        ```
+        async.api.configs:
```

Comment:
We have changed  **async.api.configs** to **service.catalog.configs** on the source, shall we do that change in the docs as well

Reply:
Fixed in ed0629a692620f8568625291553ea79661fffaed and a47e32124ad7c28853037dcd77429d2cb8df7bef